documentation/migrating/02-pkg.md

```diff
@@ -0,0 +1,20 @@
+---
+title: package.json
+---
+
+### dependencies
+
+Remove `polka` or `express`, if you're using one of those, and any middleware such as `sirv` or `compression`.
+
+### devDependencies
+
+Remove `sapper` from your `devDependencies` and replace it with `@sveltejs/kit`, `@vite`, and whichever [adapter](/docs#adapters) you plan to use (see [next section](#project-files-configuration)).
```

Comment:
I think it's just `vite` instead of `@vite`

Reply:
good catch, fixed